src/Backend/Core/Js/utils.js

```diff
@@ -303,7 +304,13 @@ utils.string =
 	 */
 	stripTags: function(value)
 	{
-		return value.replace(/<[^>]*>/ig, '');
+		if(typeof value === 'undefined') return '';
+		else
+		{
+			value = value.replace(/<[^>]*>/ig, '');
+		}
+
+		return value;
```

Comment:
I would restructure this code block this way:

```js
if (typeof value === 'undefined') {
    return '';
}

return value.replace(/<[^>]*>/ig, '');
```

This has exactly the same outcome, but is clearer imo.